src/components/AppWrapper.tsx

```diff
@@ -0,0 +1,217 @@
+import React from 'react'
+import {IInfographicsSection, TInfographics, IInfographicsRow} from '../types/IInfographicsSection'
+import Editor from './Editor';
+import Canvas from './Canvas'
+import {IInfographicsDetails} from '../types/IInfographicsDetails'
+import produce from 'immer'
+
+interface IState {
+    infographicsDetails: IInfographicsDetails,
+    lastUsedId: number,
+    selectedSection: IInfographicsSection | null,
+    columnsCount: number,
+}
+
+class AppWrapper extends React.Component<{}, IState> {
+
+    state: IState = {
+        infographicsDetails: {
+            widthPx: 400, // the default width of infographics could be set in "New infographics" form in the future
+            infographics: [] as TInfographics,
+        },
+        lastUsedId: 0,
+        selectedSection: null,
+        columnsCount: 1,
```

Comment:
nevidim nikde `setState` ktery by upravoval `columnsCount`, zda se mi to nebo to je vzdy `1`?